keylime/agentstates.py

```diff
@@ -56,6 +94,7 @@ def __init__(self, agent_id):
         self.agent_id = agent_id
         self.next_ima_ml_entry = 0
         self.set_boottime(0)
+        self.set_tpm_clockinfo({"start": True})
```

Comment:
Is there a reason to not use a `TPMClockInfo` object here?

Reply:
good catch, will change